R/authors.R

```diff
@@ -0,0 +1,103 @@
+#' Create or modify the `Authors@R` field in the DESCRIPTION file
+#'
+#' @description
+#' If the `Authors@R` field is not included in the DESCRIPTION file then `use_author()` adds the field assigns
```

Comment:
Can you use the styler package on all files or sections of files that you are touching? That will bring your formatting in line with our typical style. I'm not sure if it will deal with line length, but if it does not, note that I generally cap line length at 80 and use *Code > Reflow comment* on sections like this.

Reply:
I used `styler` through the drop down in Addins. I've never used this package so I'm not super familiar with what it should convert things to or how to check it worked. I used the default `styler::tidyverse_styel()` on all the pages I worked with. I also used Code > Reflow comment to adjust the long roxygen header at the start. I tried to adjust lines to shorter lengths where possible.